utkdigitalinitiatives/utk-fedora-fedoragsearch-solr-config
# Trac 875

Updated request.
See JIRA TRAC-875 for last two screen shots that show my testing.

This is a PULL request in supplort of JIRA TRAC-875.

Code changes are in /utk-fedora-fedoragsearch-solr-config/fedoragsearch-
transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt

BOTH PULL REQUESTS should be done before testing.
 
Testing instructions:

1.  git checkoutTRACE (TRAC-882) branch.          
2.  a. vagrant destroy
     b.  git pull --all
     c.  vagrant up.
    -- TRACE (TRAC-882) will load the utk-fedora-fedoragsearch-solr-config(TRAC-875)  
3.  userA: ingest one thesis. thesis_manager: accept-publish
    Check the url of the thesis to see the pid
4.  The new ingest should have PID = utk.ir.td:52
5.  Update the solr index so that you can find utk.ir.td:52
	a. http://localhost:8080/fedoragsearch/rest?operation=updateIndex
	b. List all pids in solr index on localhost  
http://localhost:8080/solr/collection1/select?q=*%3A*&sort=PID+asc&rows=1000000&fl=PID&wt=csv&indent=true
6.  Look at the data display at:  
    http://localhost:8080/solr/collection1/select?q=PID%3A%22utk.ir.td%3A52%22&fl=utk_mods_etd*&wt=xml&indent=true
7.  Look at the at the solr schema browser at: http://localhost:8080/solr/#/collection1/schema-browser
    See if you can locate the fieldname: utk_mods_etd_author_s



@markpbaggett 
@CanOfBees
